### ISSUE TITLE
require('graze') → errors

### ISSUE THREAD

**Issue by freder** [LukeGT/graze#1]
hi,

this library looks really nice, but `require()`ing it already gives me errors:

``` javascript
var graze = require('graze');
```

↓

```
/Volumes/Macintosh HD 2/Projects/bla/node_modules/graze/node_modules/jsdom/lib/jsdom/level1/core.js:553
      Array.prototype.splice.call(this._childNodes, refChildIndex, 0, newChild
                             ^
TypeError: Cannot set property length of [object Object] which has only a getter
    at DocumentFragment.core.Node.insertBefore (/Volumes/Macintosh HD 2/Projects/bla/node_modules/graze/node_modules/jsdom/lib/jsdom/level1/core.js:553:30)
    at DocumentFragment.<anonymous> (/Volumes/Macintosh HD 2/Projects/bla/node_modules/graze/node_modules/jsdom/lib/jsdom/level2/events.js:332:20)
    at DocumentFragment.proto.(anonymous function) [as insertBefore] (/Volumes/Macintosh HD 2/Projects/bla/node_modules/graze/node_modules/jsdom/lib/jsdom/utils.js:23:26)
    at DocumentFragment.core.Node.appendChild (/Volumes/Macintosh HD 2/Projects/bla/node_modules/graze/node_modules/jsdom/lib/jsdom/level1/core.js:671:17)
    at /Volumes/Macintosh HD 2/Projects/bla/node_modules/graze/node_modules/jquery/dist/jquery.js:4026:18
    at /Volumes/Macintosh HD 2/Projects/bla/node_modules/graze/node_modules/jquery/dist/jquery.js:4047:3
    at module.exports (/Volumes/Macintosh HD 2/Projects/bla/node_modules/graze/node_modules/jquery/dist/jquery.js:31:12)
    at Object.<anonymous> (/Volumes/Macintosh HD 2/Projects/bla/node_modules/graze/index.js:7:24)
    at Object.<anonymous> (/Volumes/Macintosh HD 2/Projects/bla/node_modules/graze/index.js:132:4)
    at Module._compile (module.js:460:26)
```

I updated the `jsdom` dependency to the latest version, but I get this now:

```
/Volumes/Macintosh HD 2/Projects/bla/node_modules/graze/node_modules/jquery/dist/jquery.js:28
                if ( !w.document ) {
                       ^
TypeError: Cannot read property 'document' of undefined
    at module.exports (/Volumes/Macintosh HD 2/Projects/bla/node_modules/graze/node_modules/jquery/dist/jquery.js:28:12)
    at Object.<anonymous> (/Volumes/Macintosh HD 2/Projects/bla/node_modules/graze/index.js:7:24)
    at Object.<anonymous> (/Volumes/Macintosh HD 2/Projects/bla/node_modules/graze/index.js:132:4)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Volumes/Macintosh HD 2/Projects/bla/test.js:14:13)
```

(and updating the rest of the dependencies, too, doesn't change anything...)

`node` v0.12.1

**Comment by luke-tsekouras-sirca** [comment]
Interesting, I'll take a look.

**Comment by LukeGT** [comment]
Sorry it took me so long to get around to this @freder. I upgraded all of graze's dependencies and refactored to suit the new APIs. I've released a new version `1.7.0` of graze which works for me on the latest node.js `0.12.1`. 

Let me know if you're still having problems.
